### PR TITLE
Omnibot ROS integration

### PR DESCRIPTION
**Pull request by Shah-Rajab**
Modifying the script to support communication with omnibot.

### CODE REVIEW

**Comment by Vuwij** [comment]
This pull request **fixes 1 alert** when merging f46721b9e6b11de0b86f336ec542a4f698e1b59d into f27fe85cd1cddb2bcaaa827eb436cd899b42c072 - [view on LGTM.com](https://lgtm.com/projects/g/utra-robosoccer/soccer_ws/rev/pr-2ecba3888f5d0758e4d280a825ec7dc190f751df)

**fixed alerts:**

* 1 for Unused import

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by Shah-Rajab** [comment]
@Vuwij hi

File: omnibot/omnibot_hardware/src/omnibot_communication/omnibot_hardware.py [hunk 1]
```diff
@@ -88,7 +81,8 @@ def __init__(self):
         self.isROSmode = args['ros']
         self.port = args['port']
         self.baud = args['baud']
-        self.dryrun = args['dryrun']
+        #self.dryrun = args['dryrun']
+        self.dryrun = False
```
[the remark this answers is not in the document]
Comment (tygamvrelis) [reply omnibot/omnibot_hardware/src/omnibot_communication/omnibot_hardware.py]: Okay, but why not just set the default value to False and then not pass it through ROS? :o

[the remark this answers is not in the document]
Comment (Shah-Rajab) [reply omnibot/omnibot_hardware/src/omnibot_communication/omnibot_hardware.py]: For some reason True was passed on, regardless of setting dryrun:=true or false

**Comment by Vuwij** [comment]
This pull request **fixes 1 alert** when merging c79d83af61e96e932bf8e2f2c7aedee6d08ab1e4 into de81d66c8a22f26d815886a2bd81eaac374d52a5 - [view on LGTM.com](https://lgtm.com/projects/g/utra-robosoccer/soccer_ws/rev/pr-25e3e3b11123bf8ed94ceee95b115a224d109460)

**fixed alerts:**

* 1 for Unused import

---

*Comment posted by [LGTM.com](https://lgtm.com)*

File: omnibot/omnibot_hardware/src/omnibot_communication/omnibot_hardware.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (Shah-Rajab) [reply omnibot/omnibot_hardware/src/omnibot_communication/omnibot_hardware.py]: @Vuwij help

**Comment by Shah-Rajab** [comment]
I think the working code is on the TX2...

**Comment by Vuwij** [comment]
This pull request **introduces 1 alert** and **fixes 6** when merging 6829b55529c1f5c62ed27a61e180d80f42155910 into ddbaaa0e18649a678a21f4ade862f8c2826967c7 - [view on LGTM.com](https://lgtm.com/projects/g/utra-robosoccer/soccer_ws/rev/pr-f0e4c4a00099875bb0f373672f72e531d177f1a8)

**new alerts:**

* 1 for Syntax error

**fixed alerts:**

* 4 for Unused import
* 2 for Except block handles &#39;BaseException&#39;

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Pull request merged**